### ISSUE TITLE
Issue when there are UTF8 characters in file names

### ISSUE THREAD

**Issue by arantes555** [InvokIT/js-untar#14]
Hello,

I'm having trouble with js-untar when trying to untar a tar file containing a single file with  a name containing accents or other special characters. More specifically, instead of finding a single file within the tar, js-untar finds 2 files with absurd names: 

```json
[
  {
    "name": "PaxHeader",
    "mode": "000666 ",
    "uid": "000000 ",
    "gid": "000000 ",
    "size": 42,
    "modificationTime": 1508509981,
    "checksum": "011124 ",
    "type": "x",
    "linkname": "",
    "ustarFormat": "ustar",
    "version": "00",
    "uname": "",
    "gname": "",
    "devmajor": "000000 ",
    "devminor": "000000 ",
    "namePrefix": "",
    "buffer": {}
  },
  {
    "name": "PaxHeader",
    "mode": "000666 ",
    "uid": "000000 ",
    "gid": "000000 ",
    "size": 2362298,
    "modificationTime": 1508509981,
    "checksum": "011033 ",
    "type": "0",
    "linkname": "",
    "ustarFormat": "ustar",
    "version": "00",
    "uname": "",
    "gname": "",
    "devmajor": "000000 ",
    "devminor": "000000 ",
    "namePrefix": "",
    "buffer": {}
  }
]
```

Don't hesitate to ask if you need any more information to help with debugging !

**Comment by arantes555** [comment]
@Qvazar Is js-untar still maintained ? Sorry to tag you but this issue is really bothering me :/

**Comment by Qvazar** [comment]
Yes it is, it's just a very busy time for me at the moment.
Hmm, seems like there's a different format for such filenames. Do you have a tar-file I can use for debugging?

**Comment by arantes555** [comment]
@Qvazar I had trouble modifying your tests to change to tar file they use, so I wrote a very simple example, that shows what js-untar gives when opening a file, with 2 tar files, one containing an empty file called `é.txt`, and one containing an empty file call `e.txt`.

Here's the link to the gist: https://gist.github.com/arantes555/496fa092b154433e695e6ca5df9e8af4

While both tar files contain only one file, js-untar shows a PaxHeader file that seems to contain extra metadata.

BTW, I think having a similar basic demo in your repo could be useful :)

Thank you, and feel free if you have any more questions

**Comment by Qvazar** [comment]
Thanks :)
It seems like there's an additional rarely used tar extension I didn't know about called POSIX.1-2001/pax (https://en.wikipedia.org/wiki/Tar_(computing)#POSIX.1-2001/pax). One of its benefits is that it supports additional character encodings for file names. 
It is actually implemented by an additional metadata file as you describe :)
I will look into implementing support for this, but cannot give an ETA yet. 
Are the affected archives still able to extract but also giving additional metadata-files, or do they just not work?

**Comment by arantes555** [comment]
@Qvazar Looks like the archive does extract but there are these metadata files. However, this breaks my use of it since in my usecase I have to have only the files I added to the tar, which don't contain this one :/

**Comment by Qvazar** [comment]
Fixed in v1.0.0.
